svf-llvm/lib/ObjTypeInference.cpp

```diff
@@ -135,6 +135,31 @@ LLVMContext &ObjTypeInference::getLLVMCtx()
  */
 const Type *ObjTypeInference::inferObjType(const Value *var)
 {
+    const Type* res = inferSingleObjType(var);
+    // infer type by leveraging the type alignment of src and dst in memcpy
```

Comment:
Best to put an example here to mention the inference based on the use at a memcpy callsite